crates/dojo-lang/src/plugin.rs

```diff
@@ -514,9 +514,12 @@ impl MacroPlugin for BuiltinDojoPlugin {
             DOJO_INTERFACE_ATTR.to_string(),
             DOJO_CONTRACT_ATTR.to_string(),
             DOJO_EVENT_ATTR.to_string(),
+            DOJO_MODEL_ATTR.to_string(),
             "key".to_string(),
             "computed".to_string(),
-            DOJO_MODEL_ATTR.to_string(),
+            // Not adding capacity for now, this will automatically
+            // makes Scarb emitting a diagnostic saying this attribute is not supported.
+            // "capacity".to_string(),
```

Comment:
I will delete all the capacity stuff in an other PR as it's legacy code and not used anymore.